### DISCUSSION TITLE
alist挂载115网盘貌似被防火墙作为攻击被拦截限制访问

### DISCUSSION THREAD

**Discussion by szhyf** [AlistGo/alist#3159] [General]
使用RailDrive 以WebDav方式挂载115网盘为本地驱动器，提供给EMBY使用，扫描媒体库文件时经常发生限制访问，信息如下，貌似被当攻击拦截，信息如下：
failed get objs: failed to list objs: <title id="title">访问被拦截！</title><script>function bindall() {var requestid = "20-31-21-20230126195520-1bf4891f";if(requestid === null || requestid === ""){return;}document.getElementById("d").innerText = requestid;}</script>
<style type="text/css"> .select {float: right;margin: 10px 20px 0 0;height: 32px;line-height: 20px;font-size: 14px;width: 80px;}</style>中文English
418
您的请求疑似攻击行为！

事件ID：False alarm ID

如果您是站长，您可以前往WAF控制台进行误报屏蔽设置，让您的访问不再被拦截
<script>function changeLanguage(value) {if (value === 'zh') {changeLanguageZh();} else if (value === 'en') {changeLanguageEn();} else {changeLanguageZh();}}function changeLanguageZh() {document.getElementById("title").innerHTML = "访问被拦截！";document.getElementById("a").innerHTML = "您的请求疑似攻击行为！";document.getElementById("b").innerHTML = "如果您是站长，您可以前往WAF控制台进行误报屏蔽设置，让您的访问不再被拦截";document.getElementById("c").innerHTML = "事件ID：";}function changeLanguageEn() {document.getElementById("title").innerHTML = "The access is blocked.";document.getElementById("a").innerHTML = "Sorry, your request has been intercepted because it appears to be an attack.";document.getElementById("b").innerHTML = "If you are the webmaster, configure related parameters on the WAF console to allow your requests.";document.getElementById("c").innerHTML = "Event ID:";}var language = navigator.language || navigator.browserLanguage;if (language.indexOf('zh') > -1) {changeLanguageZh();document.getElementById("option-zh").setAttribute("selected", true);} else if (language.indexOf('en') > -1) {changeLanguageEn();document.getElementById("option-en").setAttribute("selected", true);} else {changeLanguageZh();document.getElementById("option-zh").setAttribute("selected", true);}</script>: unexpected error

**Comment by sophronia798** [comment]
同样问题，换了二维码令牌能缓解一会，再扫描文件又弹出来，估计得找管理员解掉

**Reply by szhyf** [reply]
不用换令牌，过一会普通使用没问题，只要扫描就会大概率再出，估计是没有限制并发访问数，而115可能根据并发访问数来判断攻击的。应该增加并发访问数限制应该可以解决。这个提示的“管理员”，应该是指115网站的管理员，所以只能Alist大佬看能不能改了

**Comment by blu-jeans** [comment]
遇到同样的问题，AList挂载大量重命名的时候。 估计限制并发访问数可以解决，期待AList更新。

**Comment by Rhyme1105** [comment]
好像还没解决，今天也遇见了

**Comment by suyu0925** [comment]
有进展了吗？提供一个并发数的限制设置之类的，虽然限制了并发数会变慢，但好歹能用。只要文件夹内的文件一多，现在的状态属于不可用了

**Comment by lickceo** [comment]
这个问题解决了吗
